### PR TITLE
Fix Small Styling Errors

### PR DESCRIPTION
**Pull request by rgychiu**
Resolves #

Fixes some styling debt accumulated throughout the project and aims to finalize some styling things

### CODE REVIEW

File: frontend/src/components/Dashboard.js [hunk 1]
```diff
@@ -25,10 +25,7 @@ import {
 
 import add from '../media/add.png'
 
-import 'react-datepicker/dist/react-datepicker.css'
-import 'react-datepicker/dist/react-datepicker-cssmodules.css'
-import '../styles/index.css'
-import 'box-ui-elements/dist/preview.css'
+import '../styles/dashboard.scss'
```
Comment (choiboy98) [inline frontend/src/components/Dashboard.js]: Shouldn't we import css? Not scss?

File: frontend/src/components/DocumentClass.js [hunk 2]
```diff
@@ -17,7 +17,7 @@ import {
 import edit from '../media/greyEdit.png'
 import remove from '../media/remove.png'
 
-import '../styles/documentclasspage.css'
+import '../styles/documentclasspage.scss'
```
Comment (choiboy98) [inline frontend/src/components/DocumentClass.js]: more scss

File: frontend/src/components/DocumentClassPage.js [hunk 3]
```diff
@@ -14,9 +14,8 @@ import { getAllDocumentClasses, createDocumentClass } from '../utils/ApiWrapper'
 
 import add from '../media/add.png'
 
-import '../styles/variables.css'
-import '../styles/index.css'
-import '../styles/documentclasspage.css'
+import '../styles/variables.scss'
+import '../styles/documentclasspage.scss'
```
Comment (choiboy98) [inline frontend/src/components/DocumentClassPage.js]: more more scss

File: frontend/src/components/DocumentList.js [hunk 4]
```diff
@@ -6,8 +6,7 @@ import DocumentListItem from './DocumentListItem'
 import expand from '../media/expand.png'
 import collapse from '../media/collapse.png'
 
-import '../styles/documentlist.css'
-import '../styles/index.css'
+import '../styles/documentlist.scss'
```
Comment (choiboy98) [inline frontend/src/components/DocumentList.js]: woohooooo

File: frontend/src/components/PMMainPage.js [hunk 5]
```diff
@@ -233,7 +230,6 @@ export class PMMainPage extends Component {
           <ModalHeader>Add New Field Partner</ModalHeader>
           <ModalBody>
             <form onSubmit={this.handleNewFP}>
-              <p>Organization Name:</p>
```
Comment (arpanlaha) [inline frontend/src/components/PMMainPage.js]: why is this removed?

Comment (rgychiu) [reply frontend/src/components/PMMainPage.js]: The placeholder value is Enter Organization name here so I felt as if that label was redundant

Comment (arpanlaha) [reply frontend/src/components/PMMainPage.js]: The placeholder does disappear after you type something tho, so redundancy might be good here
